### PR TITLE
RES Max: Recovery PR

### PR DESCRIPTION
**Pull request by ZeroPointMax**
I accidentally  deleted some notes with closing #17 and merging with #18. 
Oof, Thx Github for saving my ***

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/RES/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/PRO/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/SWE/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- RES/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Review by RvNovae — APPROVED** [review]

File: RES/README.md [hunk 1]
```diff
@@ -1,25 +1,48 @@
-Betriebssystemverwaltung
-========================
-
```
Comment (Steve2955) [inline RES/README.md:3]: Ich glaube hier ist noch was schief gelaufen ^^

Comment (ZeroPointMax) [reply RES/README.md:3]: Jup, stimmt. Wäre es schlimm, wenn ich die Überschrift direkt auf main verschiebe?

Comment (ZeroPointMax) [reply RES/README.md:3]: Nevermind, habe es im anderen PR gefixt

**Pull request merged**